docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -40,17 +42,29 @@ ALTER TABLE `geneset` AUTO_INCREMENT = 1;
 4- Restart cBioPortal (restart webserver) to clean-up any cached gene lists.
 
 5- To import gene data type the following commands when in the folder `<cbioportal_source_folder>/core/src/main/scripts`:
+### Human genes
+```
+export PORTAL_HOME=<cbioportal_configuration_folder>
+export JAVA_HOME=<jre_installation_folder>
+./importGenes.pl --hgnc <gene_info.txt> --genome-build <GRCh37>
+```
+### Mouse genes
 ```
 export PORTAL_HOME=<cbioportal_configuration_folder>
 export JAVA_HOME=<jre_installation_folder>
-./importGenes.pl --genes <ncbi_species.gene_info> --gtf <gencode.v25.annotation.gtf> --genome-build <GRCh37>
+./importGenes.pl --genes <Mus_musculus.gene_info.gz> --gtf <gencode.v25.annotation.gtf> --genome-build <GRCh37>
```

Comment:
does our code really accept a gzip file? I would have thought that the user must unzip the file before running the import (and that the doc page here just omitted that step)

Reply:
@yichaoS was this a mistake?

Reply:
@sheridancbio I don't think so. We didn't update the part for importing mouse gene data at all, and the current doc for mouse gene seems to indicate the input is a gzip file: https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-gene-and-gene_alias-tables.md#mouse-genes

Reply:
@yichaoS ... yes, that is the file to download ... but unless java knows to automatically unzip a file while it is reading it [I don't think so] then the file must be manually unzipped before running the importGenes.pl command. So I think we need to change back to <Mus_musculus.gene_info> and we should probably also add a line in the instructions telling users to unzip the file. So in the "Prepare" section above, after the lines:
```
Download `Mus_musculus.gene_info.gz` from   ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Mus_musculus.gene_info.gz
```
I would add this:
```
Unzip the downloaded file with the command `gunzip Mus_musculus.gene_info.gz`
```

Reply:
@sheridancbio I added that line in this PR as you suggested. I do think tho the whole process of mouse genes import needs to be streamlined at some point. Add this .zip thing in the [issue](https://github.com/cBioPortal/cbioportal/issues/8678) I created as well.

Reply:
I'm going to merge this as-is, but I very much doubt that the command will work with the file name Mus_musculus.gene_info.gz. I'll add a comment into the docs separately with my concerns.